HealthcareSystem/PatientWebApp/wwwroot/js/patient_examinations.js

```diff
@@ -84,8 +96,61 @@ $(document).ready(function () {
     });
 
     $('#btn_close').click(function () {
+
+        var stepEvent = {
+            "startSchedulingEventId": parseInt(startEventId),
+            "userAge": parseInt(age),
+            "userGender": parseInt(gender),
+            "eventStep": parseInt(currentStep),
+            "clickEvent": parseInt(2)
+        };
+
+        $.ajax({
+            url: "/api/event/step",
+            type: 'POST',
+            contentType: 'application/json',
+            headers: {
+                'Authorization': 'Bearer ' + window.localStorage.getItem('token')
+            },
+            data: JSON.stringify(stepEvent),
+            success: function () {
+                console.log("Successful!");
+                currentStep = 0;
+            },
+            error: function () {
+                console.log("An error occurred while writing the step event");
+            }
+        });
+
         location.reload();
     });
+
+    $('#startScheduling').click(function () {
+
+        var startEvent = {
+            "userAge": parseInt(age),
+            "userGender": parseInt(gender),
+        };
+
+        $.ajax({
+            url: "/api/event/start",
+            type: 'POST',
+            contentType: 'application/json',
+            headers: {
+                'Authorization': 'Bearer ' + window.localStorage.getItem('token')
+            },
+            data: JSON.stringify(startEvent),
+            success: function (startEvent) {
+                console.log("Successful!");
+                startEventTime = startEvent.triggerTime; 
+                startEventId = parseInt(startEvent.id);
```

Comment:
Is it necessary to parseInt here and why? You already get it as a number through JSON??